lab_stack/ConsoleApplication33/ConsoleApplication33/Stack.h

```diff
@@ -24,13 +24,13 @@ class Stack
     Element* top = nullptr;
 
 public:
-    //�����������
+    //êîíñòðóêòîð
     Stack()
     {
 
     };
 
-    //����������� �����������
+    //êîíñòðóêòîð êîïèðîâàíèÿ
     Stack(const Stack& stack)
     {
         Element* element = stack.top;

```

Comment:
Delete pointer? What this pointer doing after pushing?

Reply:
Implementation must be in stack.cpp (all things in .h)

Reply:
Moved the implementation to stack.cpp. But i don't understand your question....